docs/src/components/attachments.md

```
@@ -0,0 +1,100 @@
+
+### 是否换行
+
+<demo vue="../../demos/attachments/wrap.vue" />
+
+### 自定义文件图标
+
+通过 `fileIcons` 属性可以覆盖默认的文件类型图标。
+
+<demo vue="../../demos/attachments/custom-icon.vue" />
+
+**文件卡片状态显示模式**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)